comms/dht/migrations/2020-05-14-103121_add_body_hash/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE stored_messages
+    ADD body_hash TEXT;
+
+CREATE UNIQUE INDEX uidx_stored_messages_body_hash ON stored_messages (body_hash);
```

Comment:
Tested migration and works well (nulls are not counted towards the unique index as expected)

Reply:
That is great!